tests/test_scripts.py

```diff
@@ -2175,12 +2176,21 @@ def test_sample_reads_randomly():
     outfile = infile + '.subset'
     assert os.path.exists(outfile), outfile
 
+    if sys.version_info.major == 2:
+        answer = set(['850:2:1:2691:14602/1', '850:2:1:1762:5439/1',
+                      '850:2:1:2399:20086/2', '850:2:1:2503:4494/2',
+                      '850:2:1:2084:17145/1', '850:2:1:2273:13309/1',
+                      '850:2:1:2263:11143/2', '850:2:1:1984:7162/2',
+                      '850:2:1:2065:16816/1', '850:2:1:1792:15774/2'])
+    else:
+        answer = set(['850:2:1:1946:20852/1', '850:2:1:1251:16575/1',
+                      '850:2:1:1625:9364/2', '850:2:1:3123:15968/1',
+                      '850:2:1:1601:18498/2', '850:2:1:1267:6790/1',
+                      '850:2:1:2562:16360/2', '850:2:1:1199:4197/1',
+                      '850:2:1:1832:14607/1', '850:2:1:2401:4896/1'])
```

Comment:
This is tricky. Python 3 changed the implementation of random.randrange (which underlies the random.randint implementation), so they will give different results using the same seed on Python 2 and 3. If we want to remain consistent we need to grab the old implementation and leave it somewhere in our source.